src/java.desktop/share/native/liblcms/LCMS.c

```diff
@@ -649,9 +649,22 @@ JNIEXPORT jobject JNICALL Java_sun_java2d_cmm_lcms_LCMS_getProfileID
     if (pf == NULL) {
         return NULL;
     }
-    fid = (*env)->GetFieldID (env,
-        (*env)->GetObjectClass(env, pf),
-        "cmmProfile", "Lsun/java2d/cmm/Profile;");
+
```

Comment:
The purpose of this method is to bypass java access control and access the private ICC_Profile.cmmProfile field, I have added a call for activation here as well. This could be improved someday by the accessor or something like this.